builtin/mainmenu/tab_online.lua

```diff
@@ -56,18 +56,23 @@ local function get_sorted_servers()
 end
 
 -- Persists the selected server in the "address" and "remote_port" settings
```

Comment:
This comment belongs to `set_selected_server`. Either update it (so it applies to multiple of the functions below) or move it there.